src/app/pages/news/11.3/07-form-editor.md

```diff
@@ -0,0 +1,15 @@
+## Form Editor (Alpha) {#formEditor}
+
+We are enabling you to build user interfaces like never before.
+Looking for feedback, we already integrated an early Alpha version of our new Form Editor.
+
+- **Low-Code**: Committed to let everyone partake in building UI, the editor let's you define contents without writing any code. Drag and Drop skills is all you need to start your creative journey.
+- **Agnostic**: Building the UI in technology neutral manor, we automatically generated the views for JSF as before. However, you're no longer the maintainer of it and can ignore any efforts to make it work and keep it stable.
+- **Alpha**: We plan to support much more widgets, and integrations of complex components until the LTS 12 release.
+
+<div class="short-links">
+	<a href="${docBaseUrl}/designer-guide/user-interface/user-dialogs/html-dialog-editor.html"
```

Comment:
this doc does not yet exist: I plan to initiate it, just as we did for the existing html-dialog-editor https://dev.axonivy.com/doc/11.3/designer-guide/user-interface/user-dialogs/html-dialog-editor.html

Reply:
initial docs are on the way: https://github.com/axonivy/core/pull/5748